playbooks/portals-takedown.yml

```diff
@@ -47,6 +47,7 @@
         project_src: "{{ webportal_dir }}"
         files: "{{ webportal_docker_compose_files_wanted }}"
         build: False
+        remove_orphans: True
```

Comment:
this is probably fine but is it really necessary? This might erase container data in times where we bring down a portal for debugging purposes, is that right?